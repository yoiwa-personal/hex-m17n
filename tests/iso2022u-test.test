ISO-2022-U (UTF-8 go-and-back) test.

ASCII: ABCDE
ISO8859-1: -AABCDE
ISO8859-7: -FABCDE-A
ASCII again: ABCDE
JIS x0208: $B!"!#!$(B
ASCII again: ABCDE
Now, let's goto UTF!
%GABCDE
√Å√Ç√É
ŒëŒíŒì%G
„ÄÅ„ÄÇÔºå
ABCDE
%@Back to 2022
ASCII: ABCDE
ISO8859-1: -AABCDE%@
JIS x0208: $B!"!#!$(B
Done.
