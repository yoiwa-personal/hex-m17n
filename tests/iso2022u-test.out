ISO-2022-U (UTF-8 go-and-back) test.

ASCII: ABCDE
ISO8859-1: ÁÂÃÄÅ
ISO8859-7: ΑΒΓΔΕ
ASCII again: ABCDE
JIS x0208: 、。，
ASCII again: ABCDE
Now, let's goto UTF!
ABCDE
ÁÂÃ
ΑΒΓ
、。，
ABCDE
Back to 2022
ASCII: ABCDE
ISO8859-1: ÁÂÃÄÅ
JIS x0208: 、。，
Done.
